src/test/java/com/rocana/lucene/codec/v1/TestRocanaSearchCodecV1.java

```diff
@@ -32,10 +33,53 @@
 
 /**
  * Unit test for {@link RocanaSearchCodecV1}.
+ *
+ * This test is not a fork of Lucene code.
  */
 public class TestRocanaSearchCodecV1 {
 
   /**
+   * Verify the postings format is our custom class, as we expect.
+   *
+   * We need this since our {@link RocanaPerFieldPostingsFormat} has
+   * special code that returns our real postings format even if Lucene
+   * asks the Service Provider Interface (SPI) to lookup it's own
+   * "Lucene50" postings format.
+   */
+  @Test
+  public void postingsFormatReturnsOurForkOfPerFieldPostingsFormat() {
+    RocanaSearchCodecV1 codec = new RocanaSearchCodecV1();
+
+    PostingsFormat postingsFormat = codec.postingsFormat();
+
+    Assert.assertTrue(
+      "Expected an instance of " + RocanaPerFieldPostingsFormat.class.getSimpleName(),
+      postingsFormat instanceof RocanaPerFieldPostingsFormat
+    );
```

Comment:
Do you want to use the more narrow test for equality of Class objects?  Or do you want to continue using `instanceof` so that `postingsFormat` could be a subclass of `RocanaPerFieldPostingsFormat`?

Same question for the other unit test